core/common/src/rpc/web/WebAppRpcRequest.ts

```diff
@@ -31,9 +31,11 @@ export class WebAppRpcRequest extends RpcRequest {
   private get _headers() { return this._request.headers as { [key: string]: string }; }
 
   /** The maximum size permitted for an encoded component in a URL.
+   * Note that some backends limit the total cumulative request size. Our current node backends accept requests with a max size of 16 kb.
+   * In addition to the url size, an authorization header may also add considerably to the request size.
    * @note This is used for features like encoding the payload of a cacheable request in the URL.
    */
-  public static maxUrlComponentSize = 4096;
+  public static maxUrlComponentSize = 1024 * 8;
```

Comment:
What is the current default in the Node versions we support? I know at one point it was 4096.

Reply:
The current default for the maxurlcomponentsize? I don't believe its enforced by node, unless you're referring to the total size of request headers accepted by node? Maybe I am misunderstanding the question

Reply:
Yes, I was referring to the total size of request headers accepted by node.

Reply:
It looks like Node version >14 has a max header size of 16kb. Node versions < 14 have a max header size of 8kb.